prow/plugins/approve/approve_test.go

```diff
@@ -379,11 +387,12 @@ Approvers can cancel approval by writing ` + "`/approve cancel`" + ` in a commen
 			hasLabel: true,
 			files:    []string{"a/a.go", "b/b.go"},
 			comments: []github.IssueComment{
-				newTestComment("bOb", "stuff\n/lgtm \nblah"),
+				newTestComment("bOb", "stuff\n/approve \nblah"),
```

Comment:
Why did this have to change?

Reply:
Strictly speaking, it didn't. But I didn't want other tests relying on `/lgtm` in any way, as it's now a non-default option to have it match the regex too